code/modules/sanity/sanity_lists.dm

```diff
@@ -110,10 +110,10 @@
 			"Escaping from your purpose is impossible...",
 			"Seven times seven. Seven times seven. Seven times seven. Seven times seven. Seven times seven. Seven times seven. Seven times seven. Seven times seven. Seven times seven. ",
 			"And so they laughed. They laughed. You know they laughed. You heard them laugh. They laughed. They laughed.",
-			"Whenever you see a mirror, you feel like you can just reach out� and step through it.",
+			"Whenever you see a mirror, you feel like you can just reach out� and step through it.",
```

Comment:
curious symbol here, not sure if it's an issue, but I've not seen this happen in Eris code before.